Neos.ContentRepository.Core/Tests/Behavior/Fixtures/PostalAddress.php

```diff
@@ -18,26 +18,14 @@
  *
  * @see https://schema.org/PostalAddress
  */
-final class PostalAddress
+final readonly class PostalAddress
```

Comment:
Are we on PHP 8.2+ already?

Reply:
Yes, we are.
https://github.com/neos/neos-development-collection/commit/6806f731bf72bc90057ab2be184ec9cfc2e25651

Reply:
Right, thanks! :)